ckanext/scheming/plugins.py

```diff
@@ -163,6 +165,15 @@ def update_config(self, config):
 
         self._expanded_schemas = _expand_schemas(self._schemas)
 
+        # Set to support Accordions in Edit Forms
+        if (p.toolkit.asbool(p.toolkit.config.get(CKANEXT_SCHEMING_FIELD_GROUP_FOR_PACKAGES_ENABLED, False))
+                or p.toolkit.asbool(p.toolkit.config.get(CKANEXT_SCHEMING_FIELD_GROUP_FOR_RESOURCES_ENABLED, False))):
```

Comment:
These can be replaced with `toolkit.h.get_boolean_config_option`
```suggestion
        if toolkit.h.get_boolean_config_option(CKANEXT_SCHEMING_FIELD_GROUP_FOR_PACKAGES_ENABLED)
                or toolkit.h.get_boolean_config_option(CKANEXT_SCHEMING_FIELD_GROUP_FOR_RESOURCES_ENABLED):
```